packages/example/config/terra2testnet.ts

```diff
@@ -0,0 +1,92 @@
+/**
+ * remark: This is an example config file for a terra testnet chain and it should be deleted before merging.
+ */
+import type { Chain, AssetList } from "@chain-registry/types";
+
+export const terra2testnet: Chain = {
```

Comment:
will merge this all for now, however, let's move this to the chain-registry soon